templates/experiments/chat/input_bar.html

```diff
@@ -114,19 +147,20 @@ <h4 class="font-bold">File Search Files</h4>
       message: "",
       code_interpreter_files: [],
       file_search_files: [],
+      experiment_files: [],
       total_upload_size_bytes: 0,
       disableCodeInterpreterUpload: false,
       init() {
-        let fieldsToWatch = ['message']
+        let fieldsToWatch = ['message', 'experiment_files'];
```

Comment:
This should only be added if `this.$refs.experimentUpload` exists, same pattern as the others.